contracts/Scarb.toml

```diff
@@ -26,5 +26,5 @@ sort-module-level-items = true
 
 [[tool.snforge.fork]]
 name = "Mainnet"
-url = ""
+url = "https://starknet-mainnet.g.alchemy.com/v2/v0Md_5fihoGkWsiwO1I0FFwPgFu9KVLB"
```

Comment:
boom, you leaked your private key! 😄 revoke your alchemy api access and make sure to not commit your Scarb.toml file

Reply:
yeah it is the second time i did it, already revoked once ... i'm a fool

Reply:
i think someone just spammed my API access, i just saw 1500 requests when deleting ^^